src/huggingface_hub/_space_api.py

```diff
@@ -17,6 +17,12 @@
 from typing import Dict, Optional
 
 
+# Taken from https://huggingface.co/docs/hub/spaces-gpus#sleep-time
+# This value might change in the future. It is only used to trigger a warning if the user tries to set a different value
+# on a free Space.
+CPU_BASIC_SLEEP_TIME = 48 * 3600  # 48 hours
```

Comment:
Maybe we should always send a warning no matter the value if the user tries to configure sleep time on a free space?

Reply:
Yep actually why not, that would simplify the logic :+1: